doc/man/config.rst

```diff
@@ -0,0 +1,48 @@
+Config file support
+===================
+
+Config files are supported by most subcommands of ``pkgdev`` from any of three
+locations. Listed in order of increasing precedence these include the
```

Comment:
Could you mention how to find out about more options?

Reply:
Using `pkgdev {subcommand} --help`, but how to write it?

Reply:
"To find all possible configuration options, run: pkgdev {subcommand} --help".